addons/storyshots/.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "presets": ["env", "react"],
+  "presets": ["env", "stage-0", "react"],
```

Comment:
Why is it needed? Can we include the particular transform instead?

Reply:
for the destructuring assignment here: https://github.com/storybooks/storybook/pull/2414/files#diff-75fdf0f2c6757366489b97c6898ea585R6

i would have used stage-2, but storybook itself pulls in [stage-0](https://github.com/storybooks/storybook/blob/master/.babelrc#L2), so i went with that to stay consistent

Reply:
Oh I see. I wonder if we can remove this nested `.babelrc` to avoid confusion 

@ndelangen what do you think?